aspnetcore/signalr/configuration.md

```diff
@@ -210,7 +210,7 @@ Additional options for configuring timeout and keep-alive behavior are available
 
 | .NET Option | JavaScript Option | Default Value | Description |
 | ----------- | ----------------- | ------------- | ----------- |
-| `ServerTimeout` | `serverTimeoutInMilliseconds` | 30 seconds (30,000 milliseconds) | Timeout for server activity. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). |
+| `ServerTimeout` | `serverTimeoutInMilliseconds` | 30 seconds (30,000 milliseconds) | Timeout for server activity. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). The recommended value of this setting is double the servers `KeepAliveInterval` value. |
```

Comment:
> The recommended value of this setting is double the servers `KeepAliveInterval` value.

Suggested replacement: "This value must be large enough to ensure that a ping message can be sent from the server **and** received by the client within the timeout interval. We recommend this value always be at least **double** the value of the server's `KeepAliveInterval` setting, to ensure time for pings to arrive"

(Make sure to use an `'` on `server's` since it's possessive ;))